### DISCUSSION TITLE
Is there a way of hiding the progress timer and "Deploying ..." when using CI/CD?

### DISCUSSION THREAD

**Discussion by pcolmer** [serverless/serverless#10226] [Q&A]
I've got a GitHub Actions workflow that triggers `npx serverless` to deploy the project when a change is committed to the git repo. However, because of how `serverless` displays progress normally in a terminal, i.e. <time spent> <app name> Deploying ..., the log for GitHub Actions just fills up with line after line of these updates, e.g.

![image](https://user-images.githubusercontent.com/1979931/141114101-035a3dde-6eb0-4d2d-92a0-9ca78acd20c3.png)

Is there a command-line switch I can use to turn this off?

**Comment by psoares-resilient** [comment]
https://github.com/serverless/serverless/issues/10599
